### PR TITLE
Kolla ansible dockerfile changes for 4.0 and train release

### PR DESCRIPTION
**Pull request by shyam-biradar**
All changes are related to kolla ansible
1. Added dockerfiles for train release
2. Update rpm and Debian repo urls to 4.0
3. Using single file trilio.list as repo file for ubuntu platform. Removed redundant code.

### CODE REVIEW

File: kolla-ansible/trilio-datamover-api/Dockerfile_train_centos [hunk 1]
```diff
@@ -0,0 +1,52 @@
+FROM kolla/centos-source-nova-api:train
+MAINTAINER TrilioData shyam.biradar@trilio.io
+
+LABEL name="kolla/openstack-nova-api-triliodata-plugin" \
+      maintainer="shyam.biradar@trilio.io" \
+      vendor="TrilioData" \
+      version="3.4.0" \
```
Comment (savitha-p) [inline kolla-ansible/trilio-datamover-api/Dockerfile_train_centos]: Please update the version to 4.0

File: kolla-ansible/trilio-datamover-api/Dockerfile_train_ubuntu [hunk 2]
```diff
@@ -0,0 +1,47 @@
+FROM kolla/ubuntu-source-nova-api:train
+MAINTAINER TrilioData shyam.biradar@trilio.io
+
+LABEL name="kolla/openstack-nova-api-triliodata-plugin" \
+      maintainer="shyam.biradar@trilio.io" \
+      vendor="TrilioData" \
+      version="3.4.0" \
```
Comment (savitha-p) [inline kolla-ansible/trilio-datamover-api/Dockerfile_train_ubuntu]: Please update the version to 4.0

File: kolla-ansible/trilio-datamover-api/Dockerfile_train_centos (hunk 1 above)
Comment (shyam-biradar) [reply kolla-ansible/trilio-datamover-api/Dockerfile_train_centos]: Done.

File: kolla-ansible/trilio-datamover-api/Dockerfile_train_ubuntu (hunk 2 above)
Comment (shyam-biradar) [reply kolla-ansible/trilio-datamover-api/Dockerfile_train_ubuntu]: Done.

**Pull request merged**